charts/modules/apps/mimir/values.yaml

```diff
@@ -0,0 +1,352 @@
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: target # support for enterprise.legacyLabels
+                  operator: In
+                  values:
+                    - ingester
+            topologyKey: 'kubernetes.io/hostname'
+
+          - labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/component
+                  operator: In
+                  values:
+                    - ingester
+            topologyKey: 'kubernetes.io/hostname'
+
+    zoneAwareReplication:
+      topologyKey: 'kubernetes.io/hostname'
+
+  admin-cache:
+    enabled: true
+    replicas: 2
+
+  blocks-cache:
+    enabled: true
+    replicas: 2
+
+  index-cache:
+    enabled: true
+    replicas: 3
+
+  metadata-cache:
+    enabled: true
+
+  results-cache:
+    enabled: true
+    replicas: 2
+
+  minio:
+    enabled: false
+
+  overrides_exporter:
+    replicas: 1
+    resources:
+      limits:
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+
+  querier:
+    replicas: 1
+    resources:
+      limits:
+        memory: 5.6Gi
+      requests:
+        cpu: 2
+        memory: 4Gi
+
+  query_frontend:
+    replicas: 1
+    resources:
+      limits:
+        memory: 2.8Gi
+      requests:
+        cpu: 2
+        memory: 2Gi
+
+  ruler:
+    replicas: 1
+    resources:
+      limits:
+        memory: 2.8Gi
+      requests:
+        cpu: 1
+        memory: 2Gi
+
+  store_gateway:
+    persistentVolume:
+      size: 10Gi
+    replicas: 3
+    resources:
+      limits:
+        memory: 2.1Gi
+      requests:
+        cpu: 1
+        memory: 1.5Gi
+    topologySpreadConstraints: {}
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: target # support for enterprise.legacyLabels
+                  operator: In
+                  values:
+                    - store-gateway
+            topologyKey: 'kubernetes.io/hostname'
+
+          - labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/component
+                  operator: In
+                  values:
+                    - store-gateway
+            topologyKey: 'kubernetes.io/hostname'
+    zoneAwareReplication:
+      topologyKey: 'kubernetes.io/hostname'
+
+  nginx:
+    replicas: 1
+    resources:
+      limits:
+        memory: 731Mi
+      requests:
+        cpu: 1
+        memory: 512Mi
+
+  # Grafana Enterprise Metrics feature related
+  admin_api:
+    replicas: 1
+    resources:
+      limits:
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 64Mi
+
+  gateway:
+    replicas: 1
+    resources:
+      limits:
+        memory: 731Mi
+      requests:
+        cpu: 1
+        memory: 512Mi
+  # serviceAccount:
+  #   annotations:
```

Comment:
Can you add a comment that suggests when use this service account. Ideally when mimir is deployed in EKS

Reply:
Example for these values is in values-test.yaml. So I removed it from values.yaml.